articles/container-instances/container-instances-github-action.md

```diff
@@ -121,8 +121,8 @@ az role assignment create \
 
 ### Create workflow file
 
-1. In the GitHub UI, select **Actions** > **New workflow**.
-1. Select **Set up a workflow yourself**.
+1. In the GitHub UI, select **Actions**.
+1. Select **set up a workflow yourself**.
```

Comment:
You don't see `New workflow` button if no actions are configured in the repository.

![image](https://user-images.githubusercontent.com/17608272/146910317-891e36b2-df13-4864-a7de-9d7faf5c8391.png)